TAScheduler/UnitTests/tests.py

```diff
@@ -141,40 +150,239 @@ class TestInstructorLecTAAsmgt(TestCase):
 class TestInstructorLabTAAsmgt(TestCase):
     pass
 
+
 class TestCourseInit(TestCase):
     pass
 
 
 class TestCourseAddInstructor(TestCase):  # Randall
-    pass
+    def setUp(self):
```

Comment:
You need to state all variables assigned to self outside of the methods.
```suggestion
    hold_course = None
    tempCourse = None
    instructor_user = None
    instructor_model = None
    instructor = None
    
    def setUp(self):
```